src/Pages/SingleVideo/SingleVideo.jsx

```diff
@@ -0,0 +1,185 @@
+import { useParams } from "react-router-dom";
+import { useEffect, useState } from "react";
+import { VideoPlayer } from "../../Components/VIdeoPlayer/VideoPlayer";
+import { useAuth, useAxiosCalls, useModal, useVideo } from "../../Context";
+import "./SingleVideo.css";
+import { VerticalCard } from "../../Components/Cards/VerticalCard";
```

Comment:
Just a suggestion, you can export all the components in one (index.js) file and import it from there

Reply:
Ok, will fix that.